scripts/ct/run.sh

```diff
@@ -80,6 +80,16 @@ if [ "${WHICH_APP}" = 'novalue' ]; then
     exit 1
 fi
 
+beginswith() { case $2 in "$1"*) true;; *) false;; esac; }
```

Comment:
this one is not needed

Reply:
sorry, must have forgotten to delete this line.

will fix.

Reply:
Fixed!